testutils/script.go

```diff
@@ -0,0 +1,206 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package testutils
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"slices"
+	"strings"
+	"text/tabwriter"
+	"time"
+
+	"github.com/cilium/statedb"
+	"github.com/rogpeppe/go-internal/testscript"
+	"gopkg.in/yaml.v3"
+)
+
+type Cmd = func(ts *testscript.TestScript, neg bool, args []string)
+
+func ShowTableCmd[Obj statedb.TableWritable](db *statedb.DB, tbl statedb.Table[Obj]) Cmd {
```

Comment:
Hmm, I think I'll try another variation to add completely generic statedb commands that just need `DB`, e.g.
"cmp_table services", "show_table services", "tables" (like sqlite .tables), etc.

Reply:
Done.